src/Http/Routing/test/UnitTests/Builder/EndpointRoutingApplicationBuilderExtensionsTest.cs

```diff
@@ -74,7 +74,7 @@ public async Task UseRouting_ServicesRegistered_NoMatch_DoesNotSetFeature()
         }
 
         [Fact]
-        public async Task UseRouting_ServicesRegistered_Match_DoesNotSetFeature()
+        public async Task UseRouting_ServicesRegistered_Match_DoesNotSetsFeature()
```

Comment:
Might have been a revert, but the previous name sounds like it was correct. I wouldn't block on this though.